src/main/java/org/wildfly/security/util/ServiceLoaderSupplier.java

```diff
@@ -39,7 +39,7 @@
 public class ServiceLoaderSupplier<E> implements Supplier<E[]> {
 
     private final Class<E> service;
-    private final ClassLoader classLoader;
+    protected final ClassLoader classLoader;
```

Comment:
This is an API change; I think we may want to keep this package-private unless we have a need to open it to external subclasses.